From 39f166e86bdccb33618fd3c28672c92c6d0267ca Mon Sep 17 00:00:00 2001
From: "Steven Rostedt (VMware)" <rostedt@goodmis.org>
Date: Thu, 16 Feb 2017 17:47:26 -0500
Subject: [PATCH 124/134] trace-cmd: Add a commented single quote to the
 Makefile

My emacs highlighting gets confused by the single escaped quote, and makes
the rest of the file highlighted as if the rest of the file is part of a
string. By adding a commented out single quote, my emacs editor displays
everything nicely again.

Signed-off-by: Steven Rostedt (VMware) <rostedt@goodmis.org>
---
 Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile b/Makefile
index 571af626841d..a3c6bfa3b58b 100644
--- a/Makefile
+++ b/Makefile
@@ -76,6 +76,7 @@ var_dir_SQ = '$(subst ','\'',$(var_dir))'
 
 HELP_DIR = -DHELP_DIR=$(html_install)
 HELP_DIR_SQ = '$(subst ','\'',$(HELP_DIR))'
+#' emacs highlighting gets confused by the above escaped quote.
 
 BASH_COMPLETE_DIR ?= /etc/bash_completion.d
 
-- 
2.5.5

